### PR TITLE
Willow edits

### LINKED ISSUE #12: add spectrum generation feature
Add a feature that assumes an initial population distribution and computes weighted, incoherent lorentzian spectrum from a list of transition amplitudes and the initial populations

### PR DESCRIPTION
**Pull request by ws-general**
NV ensemble spectra methods and example Jupiter Notebook complete.

### CODE REVIEW

File: spectra/nv_ground_state.py [hunk 1]
```diff
@@ -216,14 +276,107 @@ def plot_nv_ground_magnetic_transition_amplitudes(transition_bvec, static_bvec,
...
-    phi = 0. * np.pi / 180. # Polar angle
-    theta = 0.
-    static_bvec = bmag * np.array([np.sin(phi) * np.cos(theta), np.sin(phi) * np.sin(theta), np.cos(phi)])
-    transition_bvec = np.array([0., 1.E-4, 0.E-4])
-
-    plot_nv_ground_magnetic_transition_amplitudes(transition_bvec=transition_bvec,
-                                                  static_bvec=static_bvec)
-    plot_nv_ground_eigenspectrum(NVGroundParameters14N(), bvector=static_bvec)
+
+# ----------------------------------------------------------------------------------------------------------------------
```
Comment (cmordi) [inline spectra/nv_ground_state.py:346]: Possibly more Pythonic to use the set of three quotation marks instead when commenting multiple line

**Review by cmordi — APPROVED** [review]
Overall good. I did not run it locally, merely looked at the implementation and standard Python practices. The only thing worth mentioning is that you may have to decide on the standard way you would like to handle multiline comments, I assume that you commented code that you will address later differently to the docstrings as you did not want to get confused. If so, that is understandable.

File: spectra/nv_ground_state.py (hunk 1 above)
Comment (ws-general) [reply spectra/nv_ground_state.py]: Got rid of the other examples and thus anything commented in the main.

**Comment by mfparsons314** [comment]
My apologies, I don't think I pushed the last button the other day.  Let's see if this works.

**Review by mfparsons314 — CHANGES_REQUESTED** [review]
Here are some comments to address.

File: spectra/nv_ground_state.py [hunk 2]
```diff
@@ -35,12 +36,47 @@ def twonplus1(n):
     return 2 * n + 1
 
 
+def rotate_frame(bvec, nv1, nv2):
+    """
+    Expresses a lab-frame vector in the nv1 axis system (nv1 is the z-axis)
+    :param bvec: vector
+    :param nv1: NV orientation
+    :param nv2: NV conjugate
```
[the remark this answers is not in the document]
Comment (mfparsons314) [reply spectra/nv_ground_state.py]: Please fix this-- I still don't understand from your docstring the role of nv2 here.

**Review by mfparsons314 — APPROVED** [review]
These look good to me!

**Pull request merged**